### PR TITLE
begin adding githubteacher bio

### PR DESCRIPTION
**Pull request by githubteacher**
This work is related to #56

### CODE REVIEW

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# GitHub Teacher Bio
+- Name: GitHub Teacher
```
Comment (githubteacher) [inline docs/githubteacher.md]: There should be more information to add here that is found in the corresponding issue.

More coming soon!

**Comment by githubteacher** [comment]
Going to add additional work now. :stuck_out_tongue_winking_eye:

**Comment by githubteacher** [comment]
@denishpatel should be all fixed now! thanks for the review!

**Comment by githubteacher** [comment]
hey @crichID take a look at this feature, could you jump on this branch and help out? (but not really, just describing how to communicate to get some collaboration)

**Comment by githubteacher** [comment]
@denishpatel can you give me a :+1: if you think this feature is complete enough for now?

**Comment by denishpatel** [comment]
@githubteacher Looks good to me. :+1:

**Pull request merged**